frontend/pluginloader.lua

```diff
@@ -43,4 +45,28 @@ function PluginLoader:loadPlugins()
     return self.plugins
 end
 
+-- TODO: Do not use registerToMainMenu() in plugins.
+function PluginLoader:addToMenu(registered_widgets, tab_item_table)
+    local more_plugins = nil
+    for __, widget in pairs(registered_widgets) do
+        if type(widget.name) ~= "string" or G_reader_settings:nilOrTrue("preferred_" .. widget.name) then
```

Comment:
I suggest we create a configuration `preferred_plugins` that contains all plugin to be shown in the main plugin menu to keep the global setting namespace less cluttered.